### PR TITLE
Instructor help page navbar covers info #2337

### LINKED ISSUE #2337: Instructor help page navbar covers info
After clicking a link from table of contents, info is covered by navbar.

### PR DESCRIPTION
**Pull request by lejolly**
Fixes #2337

### CODE REVIEW

File: src/main/webapp/instructorHelp.html [hunk 1]
```diff
@@ -24,6 +24,10 @@
     });
     </script>
 
+    <style>
+        body {padding: 0 0 30px}
+    </style>
```
Comment (kanghj) [inline src/main/webapp/instructorHelp.html]: What is this for? Is it possible to remove this?

**Comment by kanghj** [comment]
Can you post a screenshot so @damithc can look at the changes?

**Comment by lejolly** [comment]
Old behaviour, navbar is on top of content:
teammates_online_peer_feedback_system_for_student_team_projects_-_instructor_help

New behaviour without the css <style>:
teammates_online_peer_feedback_system_for_student_team_projects_-_instructor_help

The css <style> basically overrides the padding for the body defined in /src/main/webapp/stylesheets/teammatesCommon.css
teammates_online_peer_feedback_system_for_student_team_projects_-_instructor_help

The padding above the body is not needed as the navbar is no longer fixed to the top. 

This fixes the space above the navbar:
teammates_online_peer_feedback_system_for_student_team_projects_-_instructor_help

**Comment by lejolly** [comment]
New behaviour without the css <style>: 
teammates_online_peer_feedback_system_for_student_team_projects_-_instructor_help1

The css <style> basically overrides the padding for the body defined in /src/main/webapp/stylesheets/teammatesCommon.css 
teammates_online_peer_feedback_system_for_student_team_projects_-_instructor_help2

The padding above the body is not needed as the navbar is no longer fixed to the top. 

This fixes the space above the navbar: 
teammates_online_peer_feedback_system_for_student_team_projects_-_instructor_help3

**Comment by kanghj** [comment]
Ok, looks good to me.

**Pull request merged**